### ISSUE TITLE
[UI Feature] Tooltip for some buttons when hovering over them.

### ISSUE THREAD

**Issue by ravioli0509** [benawad/dogehouse#1179]
**Is your feature request related to a problem? Please describe.**
Not a severe problem, it's more like a would-like feature. Some newcomers won't be able to understand what some buttons do. Some buttons already have labels, but the home buttons don't have any labels. I feel like there needs to be some consistency.

**Buttons With No Labels**
![スクリーンショット 2021-03-18 12 23 48](https://user-images.githubusercontent.com/46848990/111568830-ce65bf80-87e4-11eb-8d69-64f473c934bf.png)

**Buttons That Do Have Labels**
![スクリーンショット 2021-03-18 12 28 05](https://user-images.githubusercontent.com/46848990/111569133-73809800-87e5-11eb-956a-bac08a7349a4.png)

**Describe the solution you'd like**
 Up to now I have seen the React side of the source code. Possibly add kofta/src/app/components/Tooltip.tsx. 
 Labels should be dynamically translated as well.

**Describe alternatives you've considered**
Simply put a label beneath but UI wise doesn't look pretty in my opinion :(

**Additional context**
Some tooltip examples, [Material UI](https://material-ui.com/components/tooltips/).  (Doesn't have to be mui)

**Comment by Lawqup** [comment]
Hey, I think I can try and make a prototype using Material UI, but I can't seem to find where I'd implement it in the [kofta](https://github.com/benawad/dogehouse/blob/staging/kofta) file. I'm definitely new to this project so any help would be appreciated!

**Comment by suyoin** [comment]
> Hey, I think I can try and make a prototype using Material UI, but I can't seem to find where I'd implement it in the [kofta](https://github.com/benawad/dogehouse/blob/staging/kofta) file. I'm definitely new to this project so any help would be appreciated!

I believe you should be doing it in [kibbeh](https://github.com/benawad/dogehouse/blob/staging/kibbeh), the directory for the Next.js frontend.
